nucliadb_texts3/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
```

Comment:
The best way to review nucliadb_texts3 is to download this branch and diff against nucliadb_texts2: `diff -ru nucliadb_texts2/ nucliadb_texts3`

Reply:
I think we need to be able to filter by field type aswell, so maybe you could split the nucliadb-field parts into two separate tantivy fields? one for the type (a, u, t, f...) and another one for the key part

wdyt?

Reply:
Oh, I already changed it back to facets. I'm pretty sure nobody uses that feature, but well.